GSA/digitalgov.gov
# Improve a11y issues on content pages

## Summary
Updated a11y for `a` tags by removing override for text underline.


> **Note**
> Removed & Refactored unused styles in `_content.scss` & `_resources.scss` (ca27d8ef1cbf09c4344d99c5e617fa113d19d4dd)
> 

### Preview
- [Content Page Links](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-improve-a11y-content-pages/2023/06/13/how-to-choose-the-right-metrics-for-your-intranet/)

- [Resources Table](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-improve-a11y-content-pages/resources/plain-language-web-writing-tips/)

### Solution
Improve contrast for links within `usa-prose` by adding underline to links.


| Content     | AxeDevTools |
| ----------- | ----------- |
| Home Page   | Passing     |
| News        | Passing     |
| Events      | Passing     |
| Resources   | Passing     |
| Communities | Passing     |
| Authors     | Passing     |
| Topics      | Passing     |


**Axe DevTool Issue**
<img width="1680" alt="Screen Shot 2023-07-11 at 10 18 11 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/d63500c4-cf8e-4301-8d32-6ac4a3db8a9d">


### How To Test

1. Visit any pages (news, events, resources)
2. Run axe dev tools, should see no color contrast issues for `a` tags

---

### Dev Checklist
